management/azure_mgmt_service_bus/spec/queues_spec.rb

```diff
@@ -21,7 +21,6 @@
     namespace_parameters = Azure::ARM::ServiceBus::Models::SBNamespace.new
     namespace_parameters.location = 'westus'
     queue_parameters = Azure::ARM::ServiceBus::Models::SBQueue.new
-    queue_parameters.location = 'westus'
```

Comment:
any reason for removing this? Just wanted to make sure what has caused this?

Reply:
Because accroding to the Resource definition in ServiceBus queue, there is no location. https://github.com/Azure/azure-rest-api-specs/blob/current/specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json#L2503